packages/errors/src/messages.ts

```diff
@@ -402,6 +403,7 @@ export const SolanaErrorMessages: Readonly<{
         'Expected base58-encoded signature to decode to a byte array of length 64. Actual length: $actualLength.',
     [SOLANA_ERROR__KEYS__SIGNATURE_STRING_LENGTH_OUT_OF_RANGE]:
         'Expected base58-encoded signature string of length in the range [64, 88]. Actual length: $actualLength.',
+    [SOLANA_ERROR__KEYS__VERIFY_SIGNATURE_FAILURE]: 'The provided private key does not match the provided public key at address [$address].',
```

Comment:
This is the wrong error code, right?
```suggestion
    [SOLANA_ERROR__KEYS__PUBLIC_KEY_MUST_MATCH_PRIVATE_KEY]: 'The provided private key does not match the provided public key at address [$address].',
```

Reply:
Good catch, thanks!